### PR TITLE
fixed HL1 input

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

The Windows Mixed Reality APIs changed and handedness for the HoloLens One was enabled which broke our controller mappings.

- We added a new controller type `HololenOneController` for the HoloLens 1 device.
- We updated the `HololensOneController` controller to utilize the new handedness
- We updated the default controller mappings for the `HololensOneController`

# Breaking Changes

Breaking change mitigation:

- Delete the `HoloLensControllerProfile` asset
- Reimport the default controller assets

### CODE REVIEW

**Review by SimonDarksideJ — APPROVED** [review]
Tested and now working as expected on HoloLens One

**Comment by SimonDarksideJ** [comment]
@FejZa can you test this on HL2 to ensure we don't have a regression issue

**Comment by StephenHodgson** [comment]
@FejZa we had to put [the workaround](https://github.com/XRTK/WindowsMixedReality/pull/95) back in to get HoloLens 1 input working correctly again.

**Comment by FejZa** [comment]
I'll test this on HL2 today.

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Profiles/WindowsMixedRealityControllerDataProviderProfile.cs [hunk 1]
```diff
@@ -59,7 +59,8 @@ public override ControllerDefinition[] GetDefaultControllerOptions()
         {
             return new[]
                {
-                 new ControllerDefinition("HoloLensController", typeof(WindowsMixedRealityMotionController)),
+                 new ControllerDefinition(typeof(HololensOneController), Handedness.Left),
```
Comment (FejZa) [inline XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Profiles/WindowsMixedRealityControllerDataProviderProfile.cs:62]: Can we please fix "Hololens" to "HoloLens". I don't know why but i just don't like Hololens at all :D

Comment (SimonDarksideJ) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Profiles/WindowsMixedRealityControllerDataProviderProfile.cs:62]: The only reason we put the casing here to "Hololens" is to ensure Unity doesn't put a space in the name in the editor, which it does by default.  All other classes have the correct casing.

Comment (FejZa) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Profiles/WindowsMixedRealityControllerDataProviderProfile.cs:62]: Hmm ok. Not really happy that such things influnce code but not a big deal as well I guess. We should be able to specify a display name for these definitions.

**Review by FejZa — APPROVED** [review]

**Pull request merged**